ios-drag-drop.js

```diff
@@ -292,7 +293,7 @@
         evt.preventDefault();
         new DragDrop(evt,el);
```

Comment:
sure, pop in a local var if you like. V8 will realise it's cruft and optimise away.